examples/mysql-galera/pxc-node3-controller.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    name: pxc-node3 
+  name: pxc-node3 
+spec:
+  replicas: 1
+  selector:
+    name: pxc-node3 
+  template:
+    metadata:
+      labels:
+        name: pxc-node3 
+    spec:
+      containers:
+        - resources:
+            limits: 
+              cpu: 0.3
+          image: capttofu/percona_xtradb_cluster_5_6:latest
+          name: pxc-node3
+          ports:
+            - containerPort: 3306
+              hostPort: 3306
```

Comment:
[best practices](https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/config-best-practices.md)#7: Avoid hostPort.

Is there a reason you use it for node3 but not node 1 & 2?